src/main/java/org/dita/dost/module/DebugAndFilterModule.java

```diff
@@ -167,9 +161,9 @@ private void processFile(final FileInfo f) {
 
             in = new InputSource(f.src.toString());
 
-            final Serializer result = processor.newSerializer(outputFile);
+            final ContentHandler result = job.getStore().getContentHandler(outputFile.toURI());
```

Comment:
Instead of manually creating a S9api `Serializer` ask for a `ContentHandler` from the `Store`; all IO to temp goes through `Store`.